libsycl/CMakeLists.txt

```
@@ -0,0 +1,156 @@
+#===============================================================================
+# Setup Project
+#===============================================================================
+cmake_minimum_required(VERSION 3.20.0)
+
+set(LLVM_SUBPROJECT_TITLE "libsycl")
+
+set(LIBSYCL_SOURCE_DIR  ${CMAKE_CURRENT_SOURCE_DIR})
+set(LIBSYCL_BINARY_DIR  ${CMAKE_CURRENT_BINARY_DIR})
+
+set(CMAKE_CXX_STANDARD 17)
```

For consistency with other LLVM libraries:
```suggestion
set(CMAKE_CXX_STANDARD 17 CACHE STRING "C++ standard to conform to")
```